modules/nw-ovn-kubernetes-live-migration.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+// * networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ovn-kubernetes-live-migration_{context}"]
+= Migrating to the OVN-Kubernetes network plugin by using the live migration method
+
+The following procedure entails checking for egress router resources and using the live migration method to migrate from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin.
+
+.Prerequisites
+
+* A cluster has been configured with the OpenShift SDN CNI network plugin in the network policy isolation mode.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created a recent backup of the etcd database is available.
+* The cluster is in a known good state without any errors.
+* On all cloud platforms after updating software, a security group rule must be in place to allow UDP packets on port `6081` for all nodes.
+* Cluster administrators must remove any egress router resources.
+* You have reviewed the "Considerations for live migration to the OVN-Kubernetes network plugin" section of this document.
+
+.Procedure 
+
+. On the {product-title} web console, click *Observe* -> *Metrics*.
+
+. In the *Expression* box, add the following queries, then click *Add query*:
```

Comment:
live migration has pre-validate if there is egressrouter running on cluster,  So this query is no need I think ,we can remove it. 

this one can be replaced with :

if there is egressrouter feature on the cluster, when do migration, the network operator will blocked the migration and expose one error "
The cluster configuration is invalid (network type live migration is not supported for pods with "pod.network.openshift.io/assign-macvlan" annotation. Please remove all egress router pods). Use 'oc edit network.config.openshift.io cluster' to fix"

And also we can see the metric query `openshift_network_operator_live_migration_blocked`, there is `UnsupportedMACVLANInterface` from console. 

or we can query from cli console. 
```
#token=`oc create token prometheus-k8s -n openshift-monitoring`
# oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- curl -k -H "Authorization: Bearer $token" "https://thanos-querier.openshift-monitoring.svc:9091/api/v1/query?" --data-urlencode "query=openshift_network_operator_live_migration_blocked" | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   515  100   460  100    55   9583   1145 --:--:-- --:--:-- --:--:-- 10729
{
  "status": "success",
  "data": {
    "resultType": "vector",
    "result": [
      {
        "metric": {
          "__name__": "openshift_network_operator_live_migration_blocked",
          "container": "network-operator",
          "endpoint": "metrics",
          "instance": "10.0.3.18:9104",
          "job": "metrics",
          "namespace": "openshift-network-operator",
          "pod": "network-operator-5f9874cc47-676rg",
          "prometheus": "openshift-monitoring/k8s",
          "reason": "UnsupportedMACVLANInterface",
          "service": "metrics"
        },
        "value": [
          1717558701.173,
          "1"
        ]
      }
    ],
    "analysis": {}
  }
}
```